testing/src/main/java/io/quarkiverse/githubapp/testing/dsl/GitHubMockContext.java

```diff
@@ -8,10 +8,21 @@
 import org.kohsuke.github.GHTeam;
 import org.kohsuke.github.GitHub;
 
+import io.quarkiverse.githubapp.GitHubClientProvider;
 import io.smallrye.graphql.client.dynamic.api.DynamicGraphQLClient;
 
 public interface GitHubMockContext {
 
+    /**
+     * @return The mock for the application client.
+     * @see GitHubClientProvider#getApplicationClient()
+     */
+    GitHub applicationClient();
+
+    /**
+     * @return The mock for the installation client.
+     * @see GitHubClientProvider#getInstallationClient(Long)
+     */
     GitHub client(long id);
```

Comment:
Maybe change it to `installationClient` and same for the GraphQL one to be consistent?

We could deprecate the old ones with default methods for the time being and remove them in 2?

Reply:
Sure, I'll add a commit.

Reply:
Make sure you add `forRemoval = true` to be explicit.

Reply:
And of course I didn't :)

Fixed and force-pushed just now.